libs/sql-schema-describer/src/mssql.rs

```diff
@@ -317,7 +357,8 @@ impl SqlSchemaDescriber {
                 ind.is_primary_key AS is_primary_key,
                 col.name AS column_name,
                 ic.index_column_id AS seq_in_index,
-                t.name AS table_name
+                t.name AS table_name,
+                ind.filter_definition as filter_definition
```

Comment:
nitpick: the `as` is lowercased here but it's uppercase in the rest of the query